RandLAPACK/drivers/rl_bqrrp_gpu.hh

```diff
@@ -397,8 +398,9 @@ int BQRRP_GPU<T, RNG>::call(
         if(iter == 0) {
             lapack::geqrf_work_size_bytes(sampling_dimension, cols, A_sk_work, d, &d_size_geqrf, &h_size_geqrf, lapack_queue);
             d_work_geqrf = blas::device_malloc< char >( d_size_geqrf, lapack_queue );
-            std::vector<char> h_work_geqrf_vector( h_size_geqrf );
-            h_work_geqrf = h_work_geqrf_vector.data();
+            // Below shoudl not be necessary
+            //std::vector<char> h_work_geqrf_vector( h_size_geqrf );
+            //h_work_geqrf = h_work_geqrf_vector.data();
```

Comment:
Either remove this commented-out code or have the comment include an explanation for keeping the commented-out code.

Reply:
Removed